bs/src/tablecloth.mli

```diff
@@ -1527,41 +1527,345 @@ module Result : sig
+    otherwise, it returns `opt_a`.
+    
+    (Same as `or_else`.)
+    
+    ### Example
+    
+    ```reason
+    orElse(Some(11), Some(22)) == Some(22);
+    orElse(None, Some(22)) == Some(22);
+    orElse((Some(11)), None) == Some(11);
+    orElse(None, None) == None;
+    ```
+
+  *)
   val orElse : 'a option -> 'a option -> 'a option
 
+  (**
+    `or_else opt_a opt_b` returns `opt_b` if it is of the form `Some x`;
+    otherwise, it returns `opt_a`.
+    
+    (Same as `orElse`.)
+    
+    ### Example
+    
+    ```ocaml
+    orElse (Some 11) (Some 22) = Some 22
+    orElse None (Some 22) = Some 22
+    orElse (Some 11) None = Some 11
+    orElse None None = None
+    ```
+  *)
   val or_else : 'a option -> 'a option -> 'a option
 
+  (**
+    `map ~f:fcn opt` (`map(~f = fcn, opt)` in ReasonML) returns
+    `fcn x` (`fcn(x)` in ReasonML) if `opt` is of the form
+    `Some x` (`Some(x)` in ReasonML); otherwise, it returns `None`.
+    
+    ### Example
+    
+    ```ocaml
+    map ~f:(fun x -> x * x) (Some 9) = Some 81
+    map ~f:(fun x -> x * x) None = None
+    ```
+    
+    ```reason
+    map(~f = (x) => x * x, Some(9)) == Some(81)
+    map(~f = (x) => x * x, None) == None
+    ```
+  *)
   val map : f:('a -> 'b) -> 'a option -> 'b option
 
+  (**
+    `withDefault(~default = defVal, opt)` If `opt` is of the form `Some(x)`,
+    this function returns `x`. Otherwise, it returns the default value `defVal`.
+    
+    (Same as `with_default`.)
+    
+    ### Example
+    
+    ```reason
+    withDefault(~default = 99, Some(42)) == 42;
+    withDefault(~default = 99, None) == 99;
+    ```
+  *)
   val withDefault : default:'a -> 'a option -> 'a
 
+  (**
+    `with_default(~default: def_val, opt)` If `opt` is of the form `Some x`,
+    this function returns `x`. Otherwise, it returns the default value `def_val`.
+    
+    (Same as `withDefault`.)
+    
+    ### Example
+    
+    ```ocaml
+    with_default ~default:99 (Some 42) = 42
+    with_default ~default:99 None = 99
+    ```
+  *)
   val with_default : default:'a -> 'a option -> 'a
 
+  (**
+    `foldrValues(opt, xs)` When `opt` is of the form `Some(x)`,
+    this function returns a list with `x` prepended to the list
+    `xs`; otherwise, the list is returned unchanged.
+    
+    (Same as `foldr_values`.)
+    
+    ### Example
+    
+    ```reason
+    foldrValues(Some(99), [100, 101, 102]) == [99, 100, 101, 102];
+    foldrValues(None, [100, 101, 102]) == [100, 101, 102];
+    ```
+  *)
   val foldrValues : 'a option -> 'a list -> 'a list
 
+  (**
+    `foldr_values opt xs` When `opt` is of the form `Some x`,
+    this function returns a list with `x` prepended to the list
+    `xs`; otherwise, the list is returned unchanged.
+    
+    (Same as `foldrValues`.)
+    
+    ### Example
+    
+    ```ocaml
+    foldr_values (Some 99)  [100; 101; 102] = [99; 100; 101; 102]
+    foldr_values None  [100; 101; 102] = [100; 101; 102]
+    ```
+  *)
```

Comment:
Removing this.

Reply:
I am not getting rid of this, as another PR will get rid of it for us.

Reply:
I think I already merged the removal.